discord/ext/pages/pagination.py

```diff
@@ -291,14 +291,14 @@ def update_buttons(self) -> Dict:
 
         return self.buttons
 
-    async def send(self, messageable: abc.Messageable, ephemeral: bool = False) -> Union[discord.Message, discord.WebhookMessage]:
+    async def send(self, ctx: Union[ApplicationContext, Context], ephemeral: bool = False) -> Union[discord.Message, discord.WebhookMessage]:
         """Sends a message with the paginated items.
 
 
         Parameters
         ------------
-        messageable: :class:`discord.abc.Messageable`
-            The messageable channel to send to.
+        ctx: Union[:class:`~discord.ext.commands.Context`, :class:`~discord.ApplicationContext`]
+            The invocation context.
```

Comment:
Maybe specify the "command's invocation context"

Reply:
> Maybe specify the "command's invocation context"

Can you do it yourself? ~~I have no idea how to specify/describe the ctx parameter~~ :D

Reply:
Would just be `A command's invocation context`